internal/pkg/vm/actor/export.go

```diff
@@ -20,20 +20,13 @@ import (
 // TODO: the work of creating the wrapper should be ideally done at compile time, otherwise at least only once + cached
 // TODO: find a better name, naming is hard..
 // TODO: Ensure the method is not empty. We need to be paranoid we're not calling methods on transfer messages.
-func MakeTypedExport(actor exec.ExecutableActor, method string) exec.ExportedFunc {
-	f, ok := reflect.TypeOf(actor).MethodByName(strings.Title(method))
+func MakeTypedExport(actor exec.ExecutableActor, method types.MethodID) (exec.ExportedFunc, bool) {
+	fn, signature, ok := actor.Method(method)
```

Comment:
BLOCKING: There's a problem retrieving the executable function as a value rather than as a method.

The miner actor has a `Bootstrap` variable set in the executable actor's state. This permits two different types of actors to share most of the same code. Since the function is now a free function, that state does not get passed into the miner and no miner will ever act like a bootstrap miner.

This behavior appears to be under-tested. It will only appear when a bootstrap miner is asked to Seal or Post. 

There are two quick fixes:

1. Have actor.Method return a method instead of a function and pass `actor` in as the first parameter.
2. Create a new `bootstrapMiner` struct that duplicates all the `miner` code not excluded by the bootstrap flag and set it under the BootstrapMiner code cid in `actor.Builtin`.

There are other solutions including coming up with a better story around bootstrapping in general, but the suggestions above are probably the best way to move past this PR.